src/core/public/rendering/rendering_service.test.tsx

```diff
@@ -48,54 +48,55 @@ describe('RenderingService#start', () => {
 
   it('renders application service into provided DOM element', () => {
     startService();
-    expect(targetDomElement.querySelector('div.application')).toMatchInlineSnapshot(`
-              <div
-                class="application class-name"
-              >
-                <div>
-                  Hello application!
-                </div>
-              </div>
-          `);
+    expect(targetDomElement.querySelector('div.kbnAppWrapper')).toMatchInlineSnapshot(`
+      <div
+        class="kbnAppWrapper kbnAppWrapper--hiddenChrome class-name"
+      >
+        <div>
+          Hello application!
+        </div>
+      </div>
+    `);
   });
 
-  it('adds the `chrome-hidden` class to the AppWrapper when chrome is hidden', () => {
+  it('adds the `kbnAppWrapper--hiddenChrome` class to the AppWrapper when chrome is hidden', () => {
```

Comment:
This test is failing and I can't figure out this is the output. It might also be related to the previous comment about the observable.

```diff
  ● RenderingService#start › adds the `kbnAppWrapper--hiddenChrome` class to the AppWrapper when chrome is hidden

    expect(received).toEqual(expected) // deep equality

+    Expected: "kbnAppWrapper"
-    Received: "kbnAppWrapper class-name"
```